### PR TITLE
PUBDEV-9023 - document that DRF training metrics are differente to metrics run by model_performance

### PR DESCRIPTION
**Pull request by valenad1**

### CODE REVIEW

**Review by hannah-tillman — APPROVED** [review]
Several small updates -- LGTM :)

File: h2o-docs/src/product/data-science/drf.rst [hunk 1]
```diff
@@ -452,6 +452,21 @@ FAQ
 
   -  DRF needs to pass over up to 1M\*22\*250k = 5500 billion numbers per tree, and assuming 50 trees, that’s up to 275 trillion numbers, which can take a few hours
 
+-  **Why does DRF report different training metrics than** ``model_performance`` **?**
+
+  DRF estimates performance of the algorithm on OutOfBound samples (i.e. samples excluded from the sampling mechanism). Since ``sample_rate`` is less than 1 by default, calling ``model_performance`` can give a different number.
+
```
Comment (wendycwong) [inline h2o-docs/src/product/data-science/drf.rst:458]: OOB means out of bag.

Comment (wendycwong) [reply h2o-docs/src/product/data-science/drf.rst:458]: Nitpick:

Would love to add your discovery here:

DRF estimates the training metrics performance of the algorithm on OutofBag samples (i.e. samples excluded from the sampling mechanism) and not on the training dataset when sample_rate is less than 1 (by default).  A sample_rate of 1 will generate a warning.

Hence, calling model_performance and set test_data=train will estimate performance on the training dataset and will different from the training metrics.

**Review by wendycwong — CHANGES_REQUESTED** [review]

**Review by wendycwong — APPROVED** [review]

File: h2o-docs/src/product/data-science/drf.rst (hunk 1 above)
Comment (valenad1) [reply h2o-docs/src/product/data-science/drf.rst:459]: 👍

**Comment by valenad1** [comment]
@hannah-tillman can you check spelling again please?

**Pull request merged**